config/default.yml

```diff
@@ -61,9 +61,14 @@ RSpec:
       Pending:
         - pending
     Expectations:
+      - are_expected
```

Comment:
I haven’t personally used the ‘should’ syntax since RSpec 3 was released, but from the RSpec documentation it sounds like it’s not being removed, even in v4. So I guess we should support it?

@pirj I imagine you might know about official RSpec plans?

Reply:
All forms of `should` [will be removed from RSpec 4](https://github.com/rspec/rspec-core/pull/2803) with no replacement.

Reply:
Personally, I would not keep the burden of supporting `should`, as it was [soft-deprecated nine years ago and was disabled in the generated config since then](http://rspec.info/blog/2013/07/the-plan-for-rspec-3/#zero_monkey_patching_mode):
> For RSpec 3, we considered the idea of disabling the old syntax by default, forcing users to opt-in to use it.

But I still see people use it, even those who joined software development less than nine years ago :smile: